Source/cmake/WebKitCompilerFlags.cmake

```
@@ -155,6 +155,7 @@ if (COMPILER_IS_GCC_OR_CLANG)
         WEBKIT_APPEND_GLOBAL_COMPILER_FLAGS(-fno-exceptions)
         WEBKIT_APPEND_GLOBAL_CXX_FLAGS(-fno-rtti)
         WEBKIT_APPEND_GLOBAL_CXX_FLAGS(-fcoroutines)
+        WEBKIT_APPEND_GLOBAL_CXX_FLAGS(-fasynchronous-unwind-tables)
```

```suggestion
        WEBKIT_PREPEND_GLOBAL_CXX_FLAGS(-fasynchronous-unwind-tables)
```

APPEND is right for the other flags here because we don't want those to be overridden. But it's harmless to override this one.